### DISCUSSION TITLE
New Repository!

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#29] [Team Posts]
@App-Choreography/community : 
Following feedback we have been given, a new repository has been made!

[https://github.com/App-Choreography/BestGitHubProfileVote](https://github.com/App-Choreography/BestGitHubProfileVote)

- **Name of repo**: BestGitHubProfileVote
- **Purpose**: Enter your github profile readme screenshot into a league grouping. Those who wish to vote will need to fork the repository, and add a star/ increment the amount of given votes by one. After a specific time period, those who have the highest votes in each league will face eachother till there is only one profile left. The best profile!

**However, I am not sure if the name `BestGitHubProfileVote` is really catchy and short.**

## Looking forward to seeing some great names!
Thank you.

**Comment by Cyebukayire** [comment]
Wow, the name is good and specific and I would suggest **BestGithubProfile**
We just remove the word "Vote" to keep this short

**Comment by Cyebukayire** [comment]
Wha is the format of adding the profile?

**Comment by NS007-dev** [comment]
> Wow, the name is good and specific and I would suggest BestGithubProfile
> We just remove the word "Vote" to keep this short

Thank you @Cyebukayire for sharing your ideas. 
Removing the word vote would make it short and memorable, however, there is another GitHub Repo with a similar name: [Awesome-GitHub-Profiles](https://github.com/EddieHubCommunity/awesome-github-profiles) Made by the EddieHub community. 
I had taken a bit of inspiration from theirs, to use Profile README, but we don't want it to be similar to their name.

### As an example would *GitHubProfileLeague* sound more different and also specific? 
Thank you. 
I am also sketching up the format of adding the profiles and how it should look, and will share it below.

**Comment by NS007-dev** [comment]
Profile format:

## League 1:

| Name 1 | Name 2 | Name 3 |
| -------- | -------- | -------- |
| Profile Screenshot 1 | Profile Screenshot 2 | Profile Screenshot 3 |
| 🏅 6 |  🏅 13 | 🏅 4 |


## League 2:

| Name 4 | Name 5 | Name 6 |
| -------- | -------- | -------- |
| Profile Screenshot 4 | Profile Screenshot 5 | Profile Screenshot 6 |
| 🏅 2 |  🏅 5 | 🏅 11 |

## League 3:

| Name 7 | Name 8 | Name 9 |
| -------- | -------- | -------- |
| Profile Screenshot 7 | Profile Screenshot 8 | Profile Screenshot 9 |
| 🏅 0 |  🏅 4 | 🏅 7 |

**Comment by NS007-dev** [comment]
@App-Choreography/Community:

Here is the format above.
- Where `Name 1/2/3/4/5/6/7/8/9` goes, the contributor will add their profile name and link.
- Where `profile screenshot` is, the contributor will add a screenshot of their profile.
- Where the medals are `🏅` , those who vote, they will need to fork the repo, and go to the profile they want and increment the number of medals by one (by deleting the number next to the medals, and updating it by adding 1).
- There will be a time limit to add and cast your votes. For example, after 48 hours, whoever has the most votes from each league will be rearranged. Those who won league 3 will move up to league 2. Those who lost league one will move down to league 2. I will sort the orders after a specific amount of time. After maybe 5 days to a week the winner will be selected by having the most votes. 

### Winner:
- I will get a professionally made banner for them to add to their profile README. 
- They will be added to our website. 
- I will add the link to their profile to my own profile readme for others to see.

### If this is successful and busy, members who are willing to help us will pull requests and maintaining this repo, I will switch their role from a member of the organisation, to a maintainer. Therefore, you can merge successful pull requests, help others and resolve issues. 

If you all have any other suggestions, feel free to comment below! Whatever ideas we are happy with, you will be credited for! 
Thank you.
